ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -975,6 +976,165 @@ def traced_similarity_search(langchain, pin, func, instance, args, kwargs):
     return documents
 
 
+# TODO refactor some of these on_span_started/on_span_finished functions
+# that are used in other patched methods in this file into the utils module
+@with_traced_module
+def traced_chain_stream(langchain, pin, func, instance, args, kwargs):
+    integration: LangChainIntegration = langchain._datadog_integration
+
+    def _on_span_started(span: Span):
+        inputs = get_argument_value(args, kwargs, 0, "input")
```

Comment:
good question, let me explore that more. since we're not concerned with older versions of langchain/non-LCEL, i'll likely ignore those and verify that the name `input` is consistent and that it shows as the first in `args`

Reply:
yep, it looks like this named argument has been in place since the end of the `0.0.x` line, so we should be good here.